values-schema.yaml

```diff
@@ -27,17 +27,17 @@ definitions:
       groupInterval:
         default: 5m
         description: How long to wait before sending a notification about new alerts that are added to a group of alerts for which an initial notification has already been sent. (Usually ~5m or more.)
-        type: string
+        $ref: '#/definitions/wordCharacterPattern'
```

Comment:
Should we not make a `timePattern` that maps `<digit><timeunit>` so something like: `^[0-9]+[smh]$` for example?